content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -62,6 +62,7 @@ their authors, not the Kubernetes team.
 | Python               | [github.com/eldarion-gondor/pykube](https://github.com/eldarion-gondor/pykube) |
 | Python               | [github.com/mnubo/kubernetes-py](https://github.com/mnubo/kubernetes-py) |
 | Python               | [github.com/tomplus/kubernetes_asyncio](https://github.com/tomplus/kubernetes_asyncio) |
+| Python               | [github.com/fiaas/k8s](https://github.com/fiaas/k8s) |
```

Comment:
Please add this into the existing sort order (fiaas sorts after eldarion-gondor)

Reply:
Ooops, my bad. I didn't realise they were sorted. Will fix asap.

Reply:
@sftim : Fixed (after a git misfire)